### PR TITLE
SDKS-3324 Support new ReCaptcha Enterprise node

### PR DESCRIPTION
**Pull request by jeyanthanperiyasamy**
# JIRA Ticket

Please, link jira ticket here.

# Description

Briefly describe the change and any information that would help speedup the review and testing process.

# Definition of Done Checklist:

- [ ] Acceptance criteria is met.
- [ ] All tasks listed in the user story have been completed.
- [ ] Coded to standards.
- [ ] Ensure backward compatibility.
- [ ] API reference docs is updated.
- [ ] Unit tests are written.
- [ ] Integration tests are written.
- [ ] e2e tests are written.
- [ ] Functional spec is written/updated.
- [ ] Example code snippets have been added.
- [ ] Change log updated.
- [ ] Documentation story is created and tracked.
- [ ] Tech debts and remaining tasks are tracked in separated ticket(s).

### CODE REVIEW

File: FRCaptchaEnterprise.podspec [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+# Be sure to run `pod lib lint FRCaptchaEnterprise.podspec' to ensure this is a
+# valid spec before submitting.
+#
+# Any lines starting with a # are optional, but their use is encouraged
+# To learn more about a Podspec see https://guides.cocoapods.org/syntax/podspec.html
+#
+
+Pod::Spec.new do |s|
+  s.name             = 'FRCaptchaEnterprise'
+  s.version          = '4.5.0'
```
[the remark this answers is not in the document]
Comment (george-bafaloukas-forgerock) [reply FRCaptchaEnterprise.podspec:11]: For release that is

File: FRCaptchaEnterprise/FRCaptchaEnterprise/Sources/CaptchaConstant.swift [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+//  of the MIT license. See the LICENSE file for details.
+//
+
+import Foundation
+import FRAuth
+
+//  MARK: - CaptchaEnterpriseCallback
+
+public class CaptchaConstant {
+    static let type: String = "type"
```
Comment (george-bafaloukas-forgerock) [inline FRCaptchaEnterprise/FRCaptchaEnterprise/Sources/CaptchaConstant.swift:17]: Maybe add here the ` "com.google.recaptcha"` and "INVALID_CAPTCHA_CLIENT" to use in the 

 let error = RecaptchaError(domain: "com.google.recaptcha",
                                          code: .errorCodeInternalError,
                                          message: "INVALID_CAPTCHA_CLIENT")
        self.setClientError("\(error.errorCode)\(error.description)")

**Review by george-bafaloukas-forgerock — APPROVED** [review]
Minor comments

File: FRCaptchaEnterprise.podspec (hunk 1 above)
[the remark this answers is not in the document]
Comment (jeyanthanperiyasamy) [reply FRCaptchaEnterprise.podspec:11]: i will check stoyan on this

**Review by spetrov — APPROVED** [review]

**Pull request merged**